### PR TITLE
Add release notes for 3.4.24-2 data release

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.4/release-3-4-24-2.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.96](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   withdrawn superseded QuoVadis-Grid-ICA (1st gen) CA (BM)
+    -   added new trust anchor MD-Grid-CA-T for rollover of existing CA (MD)
+    -   discontinued expiring 2009 series MD-Grid-CA (MD)
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-24-2.md]: Can we capitalize these sub-bullets?

File: docs/release/3.4/release-3-4-24-2.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+-   [VO Package v86](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-86)
+    -   Retire the CIGI VO
+    -   Add backup lz VOMS Admin Server
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-24-2.md]:
```suggestion
    -   Add backup `lz` VOMS Admin Server
```

File: docs/release/3.4/release-3-4-24-2.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+------------------
+
+This release contains:
+
+-   CA Certificates based on [IGTF 1.96](http://dist.eugridpma.info/distribution/igtf/current/CHANGES)
+    -   withdrawn superseded QuoVadis-Grid-ICA (1st gen) CA (BM)
+    -   added new trust anchor MD-Grid-CA-T for rollover of existing CA (MD)
+    -   discontinued expiring 2009 series MD-Grid-CA (MD)
+-   [VO Package v86](https://github.com/opensciencegrid/osg-vo-config/releases/tag/release-86)
+    -   Retire the CIGI VO
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-24-2.md]:
```suggestion
    -   Retire the `CIGI` VO
```

File: docs/release/3.4/release-3-4-24-2.md (hunk 1 above)
Comment (timtheisen) [reply docs/release/3.4/release-3-4-24-2.md]: I just blindly copy these from the IGTF ChangeLog. This time the didn't capitalize. Fixed.

File: docs/release/3.4/release-3-4-24-2.md (hunk 2 above)
Comment (timtheisen) [reply docs/release/3.4/release-3-4-24-2.md]: Fixed

File: docs/release/3.4/release-3-4-24-2.md (hunk 3 above)
Comment (timtheisen) [reply docs/release/3.4/release-3-4-24-2.md]: Fixed

**Pull request merged**